readthedocs/rtdyml/__init__.py

```diff
@@ -12,6 +12,34 @@
     '../rtd_tests/fixtures/spec/v2/schema.yml'
 )
 
+ALL = 'all'
+
+
+def flatten(dic, keep_key=False, position=None):
```

Comment:
This is a simplified version of the function used internally by yamale p:

Reply:
You may want to add a reference (for the future) to the source code of yamale with a comment above the definition of this function pointing to the source code of the original one.